src/lib/layouts/workflow-run-layout.svelte

```diff
@@ -22,12 +23,22 @@
     workflowRun,
   } from '$lib/stores/workflow-run';
   import type { NetworkError } from '$lib/types/global';
+  import { copyToClipboard } from '$lib/utilities/copy-to-clipboard';
+  import { stringifyWithBigInt } from '$lib/utilities/parse-with-big-int';
 
   $: ({ namespace, workflow: workflowId, run: runId } = $page.params);
+  $: raw = $page.url.searchParams.has('json');
```

Comment:
Wondering if we want something slightly more descriptive here for the variable name, e.g. ⬇️ 
```suggestion
  $: showJSON = $page.url.searchParams.has('json');
```

Reply:
👍